lib/pages/settings/settings.dart

```diff
@@ -195,6 +195,13 @@ class GeneralConfigPage extends StatelessWidget {
                 ),
               ),
             ),
+            SwitchListTile.adaptive(
+              title: const Text('Open links in external application'),
```

Comment:
Nit: Maybe "Always open links in external application"?

We do currently open non-http links in external apps. This just makes it unconditional.

Reply:
Maybe name it "Use in-app browser" and have it enabled by default?